src/commands/deploy.ts

```diff
@@ -181,10 +179,21 @@ export default class Deploy extends Command {
         config.region === 'iran' ? '.iran.liara.run' : '.liara.run';
       const urlLogMessage = DEV_MODE
         ? // tslint:disable-next-line: no-http-string
-          `    ${chalk.cyan(`http://${config.app}.liara.localhost`)}`
-        : `    ${chalk.cyan(`https://${config.app}${defaultSubdomain}`)}`;
+          `    ${`http://${config.app}.liara.localhost`}`
+        : `    ${`https://${config.app}${defaultSubdomain}`}`;
       this.log(urlLogMessage);
 
+      const { domains } = await this.got(
+        `v1/domains?project=${config.app}`
+      ).json<IGetDomainsResponse>();
+      domains.map((domain) => {
```

Comment:
Please refactor this to use a `for...of` loop instead of `map`. We only use the map method to create a new array from an existing array by mapping each item to a new item.